galleries/users_explain/toolkits/axisartist.rst

```diff
@@ -341,18 +341,63 @@ On the other hand, there is a concept of "axis_direction". This is a
 default setting of above properties for each, "bottom", "left", "top",
 and "right" axis.
 
-========== =========== ========= ========== ========= ==========
-   ?           ?        left      bottom      right      top
----------- ----------- --------- ---------- --------- ----------
-axislabel   direction      '-'       '+'        '+'      '-'
-axislabel   rotation      180         0          0       180
-axislabel   va           center    top       center     bottom
-axislabel   ha           right    center      right     center
-ticklabel   direction      '-'       '+'        '+'      '-'
-ticklabels  rotation       90         0        -90       180
-ticklabel   ha           right    center      right     center
-ticklabel   va           center   baseline    center   baseline
-========== =========== ========= ========== ========= ==========
+.. list-table::
+
+  * -
+    -
+    - left
+    - bottom
+    - right
+    - top
+  * - axislabel
+    - direction
+    - '-'
+    - '+'
+    - '+'
+    - '-'
+  * - axislabel
+    - rotation
+    - 180
+    - 0
+    - 0
+    - 180
+  * - axislabel
+    - va
+    - center
+    - top
+    - center
+    - bottom
+  * - axislabel
+    - ha
+    - right
+    - center
+    - right
+    - center
+  * - ticklabel
+    - direction
+    - '-'
+    - '+'
+    - '+'
+    - '-'
+  * - ticklabels
+    - rotation
+    - 90
+    - 0
+    - -90
+    - 180
+  * - ticklabel
+    - ha
+    - right
+    - center
+    - right
+    - center
+  * - ticklabel
+    - va
+    - center
+    - baseline
+    - center
+    - baseline
+
```

Comment:
This is no improvement IMHO...

Reply:
If the code works, is the linter correct in complaining?

Reply:
As long as the tables are small (like in this example), I prefer ReST *simple tables*. Yes they are more cumbersome to update but they are much easier to read, and readability counts. It's not that we are changing these tables all the time.

Reply:
I'm unclear which on can go in and where we discuss the table format. As is, both change to list table but to different ones.

Reply:
> Because of 1, I'd rather discussions of this table move to that PR. If the table format is a blocker then I'll change it there.

I'm fine with that approach. I'm just wondering why you bothered to rewrite the old table here, when all it needs for a technical fix is replace the - by =, and for content changes, you again have the separate PR.

Reply:
Ok